fbu-app/PostSearch/PostSearchLeftViewController/PostSearchLeftViewController.m

```diff
@@ -30,8 +30,18 @@ - (void)viewDidLoad {
     [self.tableView setBackgroundColor:[UIColor clearColor]];
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
-    titlesArray = @[@"Back to roommate search",
-                    @"⚙ Settings",
+    titlesArray = @[@" ",
+                    @"Roommate Finder",
+                    @"Settings",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
+                    @" ",
```

Comment:
adding a bunch of blank cells is not a great practice because not all devices have the same dimensions